src/transformers/models/auto/tokenization_auto.py

```diff
@@ -22,7 +22,7 @@
 from ...utils import logging
 from ..bart.tokenization_bart import BartTokenizer
 from ..bert.tokenization_bert import BertTokenizer
-from ..bert_japanese.tokenization_bert_japanese import BertJapaneseTokenizer
+from ..bert_japanese.tokenization_bert_japanese import BertJapaneseTokenizer  # noqa: F401
```

Comment:
Is the import even needed now?

Reply:
yes, otherwise it's not in `globals()` when we try to init it. Otherwise we need to use another system for "classname_string" => actual class

Thoughts @sgugger?

Reply:
Using `globals` does not feel super clean to me. I'll make a PR on your branch to suggest an alternative class_name_string -> actual class.

Reply:
Sure, though I think we had this discussion once in the past 😉

Reply:
See https://github.com/huggingface/transformers/pull/8667

Reply:
For context, the reason I initially tried to use `globals()` was that I wanted the user to be able to inject any class here, even a class that's not in the library. But I think `globals()` doesn't do this anyways (you just get variables that are in the file scope, unless I'm mistaken?)